packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -875,15 +840,16 @@ export const components = ({
             lineHeight: odysseyTokens.TypographyLineHeightOverline,
             backgroundColor: odysseyTokens.HueNeutral50,
             color: odysseyTokens.TypographyColorSubordinate,
-            fontSize: odysseyTokens.TypographySizeSubordinate,
+            fontSize: "0.71428571rem",
+            textTransform: "uppercase",
 
             "&::before": {
               content: "''",
-              width: ".64em",
-              height: ".64em",
-              marginInlineEnd: odysseyTokens.Spacing1,
+              width: "0.42857143rem",
+              height: "0.42857143rem",
```

Comment:
these are _very_ specific values.. why is it like this?

Reply:
This is the size of the dot in the `<Status>` component, which should be 6px diameter (or the equivalent, if the root font size is different). Unfortunately we don't currently have a spacing token for 6px.

I chose to go with this very-specific rem value for two reasons:
1. Adding a new spacing token in between existing spacing tokens is nontrivial, and we're in the process of reworking our spacing tokens anyway to something more sensible, so I consider this a temporary solution.
2. We should avoid using pixel values in this case because this is supposed to scale with the size of the text.

I agree that this is ugly and not a good long-term solution, but I think it's better than the alternative.

Reply:
Thanks for the colour on this.
Can we leave a comment or something else so we know what the intentional size is supposed to be in px?